climpred/classes.py

```diff
@@ -385,36 +385,47 @@ def get_uninitialized(self):
         """Returns the xarray dataset for the uninitialized ensemble."""
         return self._datasets['uninitialized']
 
-    def smooth(self, smooth_kws='goddard2013'):
+    def smooth(self, smooth_kws=None, how='mean', **xesmf_kwargs):
         """Smooth all entries of PredictionEnsemble in the same manner to be
         able to still calculate prediction skill afterwards.
 
         Args:
-          xobj (xarray object):
-            decadal prediction ensemble output.
-
-        Attributes:
             smooth_kws (dict or str): Dictionary to specify the dims to
-                smooth compatible with `spatial_smoothing_xesmf`,
-                `temporal_smoothing` or `spatial_smoothing_xrcoarsen`.
+                smooth compatible with
+                :py:func:`~climpred.smoothing.spatial_smoothing_xesmf` or
+                :py:func:`~climpred.smoothing.temporal_smoothing`.
                 Shortcut for Goddard et al. 2013 recommendations:
-                'goddard2013'
-
-        Example:
-        >>> PredictionEnsemble.smooth(smooth_kws={'time': 2,
-            'lat': 5, 'lon': 4'})
-        >>> PredictionEnsemble.smooth(smooth_kws='goddard2013')
+                'goddard2013'. Defaults to None.
+            how (str): how to smooth temporally. From ['mean','sum']. Defaults to
+                'mean'.
+            **xesmf_kwargs (args): kwargs passed to
+                :py:func:`~climpred.smoothing.spatial_smoothing_xesmf`
+
+        Examples:
+            >>> PredictionEnsemble.smooth({'lead': 2, 'lat': 5, 'lon': 4'})
+            >>> PredictionEnsemble.smooth('goddard2013')
+            >>> PredictionEnsemble.smooth({'lon':1, 'lat':1}, method='patch')
+            >>> PredictionEnsemble.smooth({'lead':2}, how='sum')
         """
+        if not smooth_kws:
+            return self
         # get proper smoothing function based on smooth args
         if isinstance(smooth_kws, str):
             if 'goddard' in smooth_kws:
-                smooth_fct = smooth_goddard_2013
-                smooth_kws = {'lead': 4}  # default
+                if self._datasets['initialized'].lead.attrs['units'] == 'years':
```

Comment:
You could add this as a convenience property in the `PredictionEnsemble` class. On init:

```python
if self._datasets['initialized'].lead.attrs['units'] == 'years':
    self._is_annual_lead = True
else:
    self._is_annual_lead = False
```

Then just do `if self._is_annual_lead` here.